lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3907,6 +4007,11 @@ namespace {
       // FIXME: Poor location info.
       auto nameLoc = Impl.importSourceLoc(decl->getLocation());
 
+      if (auto method = dyn_cast<clang::CXXMethodDecl>(decl);
```

Comment:
Why do you have this change here? This already landed I think. You should rebase on top of a main branch that already had this code.